### PR TITLE
Upload tabs

### PR DESCRIPTION
**Pull request by bvibber**
https://mingle.corp.wikimedia.org/projects/wlm_android_app/cards/138

### CODE REVIEW

File: assets/www/js/app.js [hunk 1]
```diff
@@ -611,13 +620,19 @@ require( [ 'jquery', 'l10n', 'geo', 'api', 'templates', 'monuments', 'monument',
...
-		var username = api.userName;
+		var username = api.userName,
+			showCompleted = ( $( '#toggle-uploads-view' ).val() == 'complete-view' );
 		db.requestUploadsForUser( username ).done( function( uploads ) {
 			$( '#uploads-list' ).empty();
 			if( uploads.length ) {
 				var uploadsTemplate = templates.getTemplate( 'upload-list-item-template' );
 				var uploadCompleteTemplate = templates.getTemplate( 'upload-completed-item-detail-template' );
 				$.each( uploads, function( i, upload ) {
+					var completed = ( upload.completed == 'true' );
```
Comment (jdlrobson) [inline assets/www/js/app.js:631]: === ?

File: assets/www/js/app.js [hunk 2]
```diff
@@ -856,6 +871,10 @@ require( [ 'jquery', 'l10n', 'geo', 'api', 'templates', 'monuments', 'monument',
 			}, 400);
 		});
 
+		$( '#toggle-uploads-view' ).change( function() {
```
Comment (jdlrobson) [inline assets/www/js/app.js:874]: should be able to do this in a generic way - see https://github.com/brion/WLMMobile/blob/6b321409e224304f012d5abe8ac935a2c6d0191c/assets/www/js/app.js#L936

File: assets/www/index.html [hunk 3]
```diff
@@ -239,7 +243,12 @@
 	<div class="page" id="uploads-page">
 		<header class='actionbar'>
 			<a class='back' href="#"></a>
-			<h2><msg key="tabbar-uploads"></msg></h2>
+			<h2>
+			<select id="toggle-uploads-view">
```
Comment (jdlrobson) [inline assets/www/index.html:247]: see https://github.com/brion/WLMMobile/blob/6b321409e224304f012d5abe8ac935a2c6d0191c/assets/www/js/app.js#L936
should be able to give this a class toggle-page and set values to page names

**Comment by jdlrobson** [comment]
Back button not working at current time - probably best to encapsulate the upload later page in a separate screen

**Comment by jdlrobson** [comment]
(click uploads -> switch to incomplete uploads, click back - expect complete uploads but instead get welcome page)

**Comment by bvibber** [comment]
obsoleted by upload-tabs2

**Pull request closed without merging**